ibm/data_source_ibm_resource_group.go

```diff
@@ -72,12 +72,12 @@ func dataSourceIBMResourceGroupRead(d *schema.ResourceData, meta interface{}) er
 		}
 		grp, err := rsGroup.FindByName(resourceGroupQuery, name)
 		if err != nil {
```

Comment:
Can we add a condition check of len(grp) < 1 here also

Reply:
`FindByName` function is already returning checking the condition..